build/DependencyVersions.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CLI_SharedFrameworkVersion>2.0.4</CLI_SharedFrameworkVersion>
-    <CLI_MSBuild_Version>15.5.180</CLI_MSBuild_Version>
-    <ClI_MSBUILD_Runtime_Version>15.5.180.1</ClI_MSBUILD_Runtime_Version>
+    <CLI_MSBuild_Version>15.6.0-preview-000044-1264514</CLI_MSBuild_Version>
+    <CLI_MSBUILD_Runtime_Version>$(CLI_MSBuild_Version)</CLI_MSBUILD_Runtime_Version>
```

Comment:
I'm not sure I understand the divergence here; why was this (possibly?) different?

It looks like the versions of the runtime package differ between our MyGet private feed and nuget.org.

Reply:
@rainersigwald 15.5.180.1 was published because I broke 15.5.180 saving it with NuGet package explorer. It lost some metadata so I had to re-publish just that package and de-list 15.5.180.

Reply:
I see, thanks.